### PR TITLE
Redesign

### PR DESCRIPTION
**Pull request by GeoffroyBERNARD**
+ new logo
+ flex navbar
+ style for inputs (buttons)
+ added background image for body & nav
+ added font Coiny
+ button "traiter" is faded while not useful
+ added a separation line when result appears

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -17,13 +22,68 @@ export default {
 }
 </script>
 
+
 <style>
```
Comment (floo51) [inline src/App.vue]: Can you read [this](https://vue-loader.vuejs.org/guide/scoped-css.html) documentation ?

File: src/App.vue [hunk 2]
```diff
@@ -17,13 +22,68 @@ export default {
...
+
+@font-face {
+  font-family: "Coiny";
+  src: url('https://fonts.googleapis.com/css?family=Coiny');
+}
+
+body{
+
```
Comment (floo51) [inline src/App.vue]: Extra space

File: src/App.vue [hunk 3]
```diff
@@ -17,13 +22,68 @@ export default {
...
   color: #2c3e50;
-  margin-top: 60px;
 }
+
+input, button, .form-submit-button{
+  font-family: 'Coiny', cursive;
+}
+
+.slogan{
+  font-style:italic;
```
Comment (floo51) [inline src/App.vue]: missing space : `font-style: italic;`
You should probably add a linter. See [prettier](https://github.com/prettier/prettier)

File: src/components/Postiterie.vue [hunk 4]
```diff
@@ -73,11 +83,12 @@ export default {
 
     /*
       FUNCTION previewImage
-      preview the uploaded image
+      preview the uploaded image (and activate the button to track)
       IN : fieldName : the fieldName ;)
            file : the image
     */
     previewImage: function(fieldName, file) {
+            document.getElementById("startTracking").classList.remove("hidden");
```
Comment (floo51) [inline src/components/Postiterie.vue]: DOM direct manipulation is strongly discouraged if you're using a front-end framework like Vue, Angular or React.

**Comment by GeoffroyBERNARD** [comment]
- Added scoped style
- Fixed space issues, standardized indent
- Removed useless css and console.log 
- Removed dom manipulation / now using vue rendering (for, bind)

**Pull request merged**